setup.py

```diff
@@ -60,4 +60,7 @@ def make_distribution(self):
         'numpy',
         'sgp4>=2.2',
         ],
+    extras_require={
```

Comment:
For now let's avoid changing `setup.py`. If we tell users in the documentation that they need to install Pandas and `pyongc`, then I think that will be enough to get them going if their project needs NGC objects.